### ISSUE TITLE
[BUG]:  3.0rc1 logs.log file goes to 10 gb- WARNING logs: 2022-06-06 16:44:16.231 INFO logs: shap: 0.40.02022-06-06 16:44:16.232

### ISSUE THREAD

**Issue by tugra-alp** [pycaret/pycaret#2635]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [ ] I have confirmed this bug exists on the develop branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@develop).

### Issue Description

I have installed pycaret 3.0 rc version. I m running on py 3.7 on centos 8. When i run setup(data) function, i see full of warnings as below in the logs.log file. And the log file size reaching too almost 10 gb. 

_WARNING logs: 2022-06-06 16:44:16.231 INFO logs: shap: 0.40.02022-06-06 16:44:16.232 WARNING logs: 2022-06-06 16:44:16.231 INFO logs: shap: 0.40.02022-06-06 16:44:16.232 WARNING logs: 2022-06-06 16:44:16.231 INFO logs: shap: 0.40.02022-06-06 16:44:16.232_

The code that i am trying to run:

from pycaret.classification import *
from pycaret.datasets import get_data
data = get_data('juice')
s = setup(data, target = 'Purchase')

### Reproducible Example

```python
from pycaret.classification import *
from pycaret.datasets import get_data
data = get_data('juice')
s = setup(data, target = 'Purchase')
```

### Expected Behavior

<html>
<body>

0 | session_id | 3075
-- | -- | --
Target | Number of airline passengers
Approach | Univariate
Exogenous Variables | Not Present
Original data shape | (144, 1)
Transformed data shape | (144, 1)
Transformed train set shape | (141, 1)
Transformed test set shape | (3, 1)
Rows with missing values | 0.0%
Fold Generator | ExpandingWindowSplitter
Fold Number | 3
Enforce Prediction Interval | False
Seasonal Period(s) Tested | 12
Seasonality Present | True
Seasonalities Detected | [12]
Primary Seasonality | 12
Target Strictly Positive | True
Target White Noise | No
Recommended d | 1
Recommended Seasonal D | 1
Preprocess | False
CPU Jobs | -1
Use GPU | False
Log Experiment | False
Experiment Name | ts-default-name
USI | 4ae1

</body>
</html>

### Actual Results

```python-traceback
No errors on the terminal. Log file exhausts all the ram on the workstation
```

### Installed Versions

System:
    python: 3.7.9 (default, Mar 28 2021, 23:25:20)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
executable: /opt/jupyterhub/bin/python3.7
   machine: Linux-4.18.0-240.15.1.el8_3.x86_64-x86_64-with-centos-8.3.2011

PyCaret required dependencies:
                 pip: Installed but version unavailable
          setuptools: 62.3.2
             pycaret: 3.0.0.rc1
             ipython: Installed but version unavailable
          ipywidgets: Installed but version unavailable
                tqdm: Installed but version unavailable
               numpy: 1.21.6
              pandas: 1.3.5
              jinja2: Installed but version unavailable
               scipy: 1.7.3
              joblib: 1.1.0
             sklearn: 1.0.2
                pyod: Installed but version unavailable
            imblearn: Installed but version unavailable
   category_encoders: Installed but version unavailable
            lightgbm: Installed but version unavailable
               numba: Installed but version unavailable
            requests: Installed but version unavailable
          matplotlib: Installed but version unavailable
          scikitplot: Installed but version unavailable
         yellowbrick: Installed but version unavailable
              plotly: Installed but version unavailable
             kaleido: Installed but version unavailable
         statsmodels: Installed but version unavailable
              sktime: Installed but version unavailable
               tbats: Installed but version unavailable
            pmdarima: Installed but version unavailable
              psutil: 5.9.1

PyCaret optional dependencies:
                shap: Installed but version unavailable
           interpret: Installed but version unavailable
                umap: Installed but version unavailable
    pandas_profiling: Installed but version unavailable
  explainerdashboard: Installed but version unavailable
             autoviz: Installed but version unavailable
           fairlearn: Installed but version unavailable
             xgboost: Installed but version unavailable
            catboost: Installed but version unavailable
              kmodes: Installed but version unavailable
             mlxtend: Installed but version unavailable
       statsforecast: Installed but version unavailable
        tune_sklearn: Installed but version unavailable
                 ray: Installed but version unavailable
            hyperopt: Installed but version unavailable
              optuna: Installed but version unavailable
               skopt: Installed but version unavailable
              mlflow: Installed but version unavailable
              gradio: Installed but version unavailable
             fastapi: Installed but version unavailable
             uvicorn: Installed but version unavailable
              m2cgen: Installed but version unavailable
           evidently: Installed but version unavailable
                nltk: Installed but version unavailable
            pyLDAvis: Installed but version unavailable
              gensim: Installed but version unavailable
               spacy: Installed but version unavailable
           wordcloud: Installed but version unavailable
            textblob: Installed but version unavailable
               fugue: Installed but version unavailable
           streamlit: Installed but version unavailable
             prophet: Installed but version unavailable

**Comment by ngupta23** [comment]
@tugra-alp Could you provide the right code to reproduce this? The output seems to be coming from time series, but your code is for classification. Thanks!

**Comment by tugra-alp** [comment]
> @tugra-alp Could you provide the right code to reproduce this? The output seems to be coming from time series, but your code is for classification. Thanks!

Sorry, I was trying to run for both modules of pycaret. I got same problem for all modules. The right code is below.

from pycaret.time_series import *
from pycaret.datasets import get_data
data = get_data('airline')
s = setup(data,fh=3)

**Comment by Yard1** [comment]
Hey @tugra-alp we just pushed RC2 that should fix this issue. Could you install with `pip install -U --pre pycaret` and try again?

**Comment by ngupta23** [comment]
Thanks @Yard1 . This seems to have been fixed. I see a small log file size now.

https://gist.github.com/ngupta23/39a3f133fc2e5a03479d210a1c2900dd 

![image](https://user-images.githubusercontent.com/33585645/172268051-4f3c958f-d2cc-4b15-9cfc-bb000e44ce46.png)

**Comment by ngupta23** [comment]
@tugra-alp This seems to be fixed now. We will close this for now. Feel free to reopen it if the issue still exists on your end.
